serverless/functions/reference-content/functions-handlers.mdx

```diff
@@ -0,0 +1,39 @@
+---
+meta:
+  title: Functions handlers reference
+  description: This page explains how functions handlers work
+content:
+  h1: Functions handlers reference
+  paragraph: This page explains how functions handlers work
+tags: serverless functions cron crontab schedule cronjob
+dates:
+  validation: 2024-04-09
+  posted: 2023-04-09
+categories:
+  - serverless
+  - functions
+---
+
+## Node, Python, PHP
```

Comment:
Have 5 different tabs 🙏 

* For Node, just replace `.py` with `.js`
* For PHP, just replace `.php` with `.php`
* For rust: like go but with `.rs`